packages/core/src/libraries/protected-app.ts

```diff
@@ -113,18 +113,38 @@ export const createProtectedAppLibrary = (queries: Queries) => {
       return;
     }
 
+    const { customDomains, ...rest } = protectedAppMetadata;
+
+    const siteConfigs = {
+      ...rest,
+      sdkConfig: {
+        appId: id,
+        appSecret: secret,
+        endpoint: EnvSet.values.endpoint.href,
+      },
+    };
+
+    // Update default host (subdomain of the default domain)
     await updateProtectedAppSiteConfigs(
       protectedAppConfigProviderConfig,
       protectedAppMetadata.host,
       {
-        ...protectedAppMetadata,
-        sdkConfig: {
-          appId: id,
-          appSecret: secret,
-          endpoint: EnvSet.values.endpoint.href,
-        },
+        ...siteConfigs,
+        host: protectedAppMetadata.host,
```

Comment:
siteConfigs should includes host key?

Reply:
Yes, although this is a redunctunt value, just to keep the object type.